src/System.Net.HttpListener/src/PinvokeAnalyzerExceptionList.analyzerdata

```diff
@@ -0,0 +1,46 @@
+sspicli.dll!EncryptMessage
```

Comment:
I was trying to sort all these files for our merging ease (also remove any dupes but I don't see any)